### ISSUE TITLE
Problems with boxes.el in xemacs

### ISSUE THREAD

**Issue by carnil** [ascii-boxes/boxes#19]
Hi!

I got another bugreport for boxes in Debian, reported as https://bugs.debian.org/807439 . I'm no (x)emacs user myself, so would appreciate if you can look at it as well and confirm if this is the right overall fix to apply (if so attached to the Debian bug is a patch)

```
From: Kevin Ryde <user42_kevin@yahoo.com.au>
To: submit@bugs.debian.org
Subject: boxes: boxes.el in xemacs
Date: Wed, 09 Dec 2015 08:35:12 +1100

Package: boxes
Version: 1.1.2-3
Severity: normal
Tags: patch
File: /usr/share/emacs/site-lisp/boxes.el

boxes.el doesn't work in xemacs21 (which is still in debian) due to a
6-arg call to shell-command-on-region.

    M-x boxes-create
    => Wrong number of arguments: shell-command-on-region, 6

I suggest omitting this arg since its `nil' there is the default anyway
in gnu emacs.

[boxes.el.shell-command.diff (text/x-diff, inline)]
--- boxes.el.orig	2015-01-17 08:16:13.000000000 +1100
+++ boxes.el	2015-12-09 08:27:01.988263968 +1100
@@ -117,7 +117,7 @@
 		     (concat boxes-args " -r "))
 		 (if type
 		     (concat boxes-args " -d " type)))))
-    (shell-command-on-region start end command-string nil 1 nil)))
+    (shell-command-on-region start end command-string nil 1)))
 
 (provide 'boxes)
 ;;; boxes.el ends here
```

Regards,
Salvatore

**Comment by tsjensen** [comment]
@zerotao Since you were so quick to handle the last emacs-related issue, would you look at this one also? We greatly appreciate your help!

**Comment by zerotao** [comment]
Sure, will take a look shortly.

**Comment by tsjensen** [comment]
Fixed by @zerotao in #20 - thanks!
